Set curie as the default font instead of upstream's Liberation Mono.

media-fonts/curie is pulled in as a dependency by this ebuild when this
patch is applied (see PATCHES/RDEPEND in the .ebuild).

--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "curie:pixelsize=12:antialias=true:autohint=true";
 #if FONT2_PATCH
 /* Spare fonts */
 static char *font2[] = {
